src/style.css

```
@@ -1,3 +1,119 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: #fff;
+}
+
+.card__image {
+  display: block;
+  margin: 0 auto 40px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin: 0 0 4px;
```

1. reset default margin for `p` and `h2` tags as well
2. use margin-bottom
```suggestion
  margin-bottom: 4px;
```